tests/DependencyInjection/SchebTwoFactorExtensionTest.php

```diff
@@ -491,6 +515,109 @@ private function getFullConfig(): array
         return $parser->parse($yaml);
     }
 
+    private function getEnvVarBasedFullConfig(bool $truthyConfig): array
+    {
+        if (true === $truthyConfig) {
+            $yaml = <<<EOF
```

Comment:
You don't need a full configuration for the test. Just setting the `enabled` nodes is good enough.